src/main/java/com/spikes2212/command/genericsubsystem/GenericSubsystem.java

```diff
@@ -26,29 +37,17 @@ public GenericSubsystem() {
 
     /**
      * Constructs a new instance of GenericSubsystem with the given minSpeed and maxSpeed.
-     *
-     * @param minSpeed the minimum speed
-     * @param maxSpeed the maximum speed
      */
     public GenericSubsystem(double minSpeed, double maxSpeed) {
```

Comment:
move this constructor above the one that doesn't get anything in his signature

Reply:
לא ממש הבנתי למה הכוונה, יש מעל זה רק קונסטרקטור אחד אז אני מניח שאליו התכוונת. כשאני עושה את זה יש שגיאה. למה התכוונת?